test_runner/regress/test_relations.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from fixtures.neon_fixtures import (
+    NeonEnvBuilder,
+)
+
+
+def test_pageserver_reldir_v2(
+    neon_env_builder: NeonEnvBuilder,
+):
```

Comment:
There's a couple of cases I don't think we exercise here: listing relations, and deleting a v2 relation.